README.md

```diff
@@ -1,5 +1,9 @@
 # fira
 
+## Development Guide
```

Comment:
There is a section for "development guide" directly under this section at the same heading level

Reply:
My mistake, meant to put "Community" - fixed it in my commit